pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/template/dsl/TemplateTableTest.kt

```diff
@@ -16,11 +16,8 @@ class TemplateTableTest {
 
         val doc = outlineTestTemplate {
```

Comment:
Jeg har oppdaget at det er lettere å opprette scope direkte istedenfor å gå gjennom createTemplate. I dette tilfellet ville det blitt som følgende. Siden dette krever endringene jeg har gjort (kommer snart PR på det), så sier jeg dette mest for fremtiden.
```
val actual = OutlineScope<BokmalLang, Unit> {
   // mal-innholdet som skal testes
}
```